rpms/openshift-clients.yml

```diff
@@ -9,6 +9,7 @@ name: openshift-clients
 owners:
 - aos-master@redhat.com
 distgit:
-  # this is needed for RHEL 7 worker node support
-  # aos-cd-jobs' el8-rebuilds causes this to be rebuilt for RHEL 8 for RHCOS
   branch: rhaos-{MAJOR}.{MINOR}-rhel-7
```

Comment:
@sosiouxme if this is replacing the el8-rebuilds job, then I'm wondering if it makes more sense to remove the `distgit:branch` override and rely on the default (which is rhel-8) in 4.6 and later.